requirements/customer/glossary.tex

```diff
@@ -0,0 +1,49 @@
+\newglossaryentry{job}
+{
+  name=job,
+  plural=jobs,
+  description={
+      A job in the context of JobAdder consists of the following:
+      \begin{itemize}
+        \item A program to be executed.
+        \item A unix user who owns the job.
+        \item Priority of the task.(TODO: define priorities in user client and link them here, Urgent, High, Normal, Low)
+        \item Number of CPU threads needed for the execution of the program.
+        \item Amount of RAM needed for the execution of the program.
+        \item Paths in the network storage needed for the application.
+        \item One of:
+        \begin{itemize}
+          \item A Dockerfile providing instructions to build program environment.
+          \item The name of predefined container environment (TODO: link to list of environments we want to provide, in the WMC)
+        \end{itemize}
+     \end{itemize}
+  }
+}
+\newglossaryentry{job status}
+{
+  name=job status,
+  plural=job status,
+  description={
+    The status of a job can be:
+    \begin{itemize}
+      \item Queued - the job is waiting for a suitable work machine to become available.
+      \item Running - the job is running on some work machine.
+      \item Paused - the job has been running, but has been paused to allow for execution of other tasks.
+      \item Done - the job has finished running.
+    \end{itemize}
+  }
+}
+\newglossaryentry{job priority}
+{
+  name=job priority,
+  plural=job priorities,
+  description={
+    The priority of a job can be:
+    \begin{itemize}
+      \item Low - the job should be executed only when a server is going to be idle otherwise.
+      \item Normal - the job should be executed as soon as there is a suitable server and no tasks of higher importance.
+      \item High - the job should be executed as soon as possible, except if all work machines are already busy with Urgent jobs.
+      \item Urgent - the job should be executed immediately, even if this means pausing other tasks.
```

Comment:
This makes it sound like only urgent tasks will pause other tasks.
I think we should not define prremption logic in the glossary in the first place.
I think it's enough if we specify that there are four levels of priority and how they are ordered.